docker/worker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#set -e
```

Comment:
Don't add out-commented code. It is commonly a good idea to actually enable errors.